### ISSUE TITLE
Legg inn passport 

### ISSUE THREAD

**Issue by kennethlynne** [EdtechFoundry/edtech-seed-project-api#3]
Få opp autentiserings API for å kunne logge inn med tredjeparts tjenester (google, facebook og github) https://github.com/strongloop/loopback-example-passport

**Comment by AlmaasAre** [comment]
Yes! Bør også se på hvordan vi kan bruke angularsdk til Auth FE.

**Comment by kennethlynne** [comment]
Well well. https://stormpath.com/blog/introducing-loopback-support/
